package.json

```diff
@@ -7,8 +7,7 @@
   },
   "private": true,
   "workspaces": [
-    "packages/*",
-    "examples/*"
```

Comment:
I spent an inordinate amount of time trying to resolve typescript and webpack issues coming from adding the ssx-notepad example and making some changes. I removed the examples from being built here so that I could move forward. Since we are migrating them out of the monorepo, I assumed this is fine.